src/gui/dialogs/migrate_version_selection.cpp

```diff
@@ -0,0 +1,130 @@
+/*
+   Copyright (C) 2008 - 2018
```

Comment:
Either drop the years entirely, or put 2021 in here.

Reply:
Updated.  The years from my understanding are technically required, unfortunately, otherwise it'd be nice to not have them there at all.